quinn-proto/src/connection/mod.rs

```diff
@@ -2565,15 +2684,17 @@ where
     }
 
     /// Issue an initial set of connection IDs to the peer
-    fn issue_cids(&mut self) {
+    fn issue_cids(&mut self, now: Instant) {
         if self.local_cid_len == 0 {
             return;
         }
 
         // Subtract 1 to account for the CID we supplied while handshaking
         let n = self.peer_params.issue_cids_limit() - 1;
         self.endpoint_events
-            .push_back(EndpointEventInner::NeedIdentifiers(n));
+            .push_back(EndpointEventInner::NeedIdentifiers(now, n));
+        self.cids_issued += 1;
+        self.cids_active_seq.insert(0);
```

Comment:
Were these two lines mistakenly introduced during the rebase?

Reply:
Ahh yes. My mistake.